### PR TITLE
Update JVM args for starburstdata/presto docker image (v3)

### PR DESCRIPTION
**Pull request by findepi**
Itemized changes:

* Remove flags related to G1GC -- G1 doesn't make sense for such a small
  heap

* Remove `-XX:+HeapDumpOnOutOfMemoryError` -- while this is cool flag to
  have, it's not that useful in a docker container

### CODE REVIEW

File: presto/etc/jvm.config [hunk 1]
```diff
@@ -1,9 +1,5 @@
 -server
 -Xmx1G
--XX:+UseG1GC
```
Comment (ilfrin) [inline presto/etc/jvm.config]: but we wanted to run with the recommended settings anyway

File: presto/etc/jvm.config [hunk 2]
```diff
@@ -1,9 +1,5 @@
...
--XX:+UseGCOverheadLimit
--XX:+ExplicitGCInvokesConcurrent
--XX:+HeapDumpOnOutOfMemoryError
```
Comment (ilfrin) [inline presto/etc/jvm.config]: this was useful when running tests locally (reproducing a issue)

File: presto/etc/jvm.config (hunk 1 above)
Comment (findepi) [reply presto/etc/jvm.config]: But this looks simply funny together with `-Xmx1G`. We recommend G1GC for real cluster, not a Raspberry PI-sized docker container.

File: presto/etc/jvm.config (hunk 2 above)
Comment (findepi) [reply presto/etc/jvm.config]: Even within a docker container? I can restore.

File: presto/etc/jvm.config (hunk 1 above)
Comment (ilfrin) [reply presto/etc/jvm.config]: I understand that, we generally recomment more Xmx than 1G too, but why change other knobs if we just change one for the sake of testing, the less we change from default the better, I say

File: presto/etc/jvm.config (hunk 2 above)
Comment (ilfrin) [reply presto/etc/jvm.config]: yup

**Pull request closed without merging**